samples/drivers/crypto/sample.yaml

```diff
@@ -17,7 +17,8 @@ tests:
         - "\\[general\\] \\[INF\\] ccm_mode: CCM mode ENCRYPT - Match"
         - "\\[general\\] \\[INF\\] ccm_mode: CCM mode DECRYPT - Match"
   test-micro:
-    tags: micro
+    #FIXME fails in qemu
+    platform_exclude: qemu_nios2
```

Comment:
this patch needs to be a REVERTME with a bug filed on it, this could be a symptom of something serious (or not)